src/pages/SearchDetails/components/SearchItemPC.tsx

```diff
@@ -0,0 +1,50 @@
+import styles from 'pages/SearchDetails/SearchDetails.module.scss';
+import { Link } from 'react-router-dom';
+import { getMockItem, SHOPS } from '../static/mockup';
+
+interface Props {
+  shop: {
+    address: string,
+    placeName: string,
+    shopId: number,
+  },
+}
+
+export default function SearchItemPC({ shop }: Props) {
+  const { placeName, address } = shop;
+  const {
+    imageAlt, defaultImage, phoneNumber, image,
+  } = getMockItem();
+  return (
+    <Link to="/" className={styles.item}>
+      <div className={styles.image}>
+        <img className={styles.image__main} alt={imageAlt} src={defaultImage} />
+        <div className={styles.image__other}>
+          <img className={styles['image__other--second']} alt={imageAlt} src={image ?? defaultImage} />
+          <img className={styles['image__other--third']} alt={imageAlt} src={image ?? defaultImage} />
+          <img className={styles['image__other--fourth']} alt={imageAlt} src={image ?? defaultImage} />
+        </div>
+      </div>
+      <div className={styles.item__content}>
+        <section className={styles.item__name}>
+          <h1 className={styles.item__title}>{placeName}</h1>
+          <h2 className={styles.item__address}>{address}</h2>
+        </section>
+        <section className={styles.item__info}>
+          <div>
+            <div className={styles.item__status}>
+              <span className={styles['item__status--open']}>
+                {SHOPS.status}
+              </span>
+              <span className={styles.item__closing}>{SHOPS.closing}</span>
+            </div>
+            <div className={styles.item__distance}>{SHOPS.distance}</div>
+            <a href="tel:010-0000-0000" className={styles.item__phone}>
+              {`전화하기    ${phoneNumber}`}
```

Comment:
전화번호 링크가 잘못 걸린 것 같죠?
```suggestion
            <a href={`tel:${phoneNumber}` className={styles.item__phone}>
              {`전화하기    ${phoneNumber}`}
```

Reply:
링크 href 값 수정하고 기존에 카드를 클릭하면 바로 이동하도록 변경해서 전화번호 클릭 시 전화 기능이 작동하지 않는 현상도 해결했습니다.
원인: 이벤트 버블링
[2f8ba7f](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/54/commits/2f8ba7f621ed9d89886bf3435639b74fa7784363)